Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/NetworkSearchController.java

```diff
@@ -0,0 +1,92 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.profile.ISearchResult;
+import edu.asu.spring.quadriga.profile.IService;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveJsonProjectManager;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by vikranthdoosa on 2/19/16.
+ */
+@Controller
+public class NetworkSearchController {
```

Comment:
Done. This is very confusing as some files have indentation of a tab and others have 4 spaces. I think 4 spaces is widely used. It would be helpful to all of us if you can talk about coding styles in the class.